ddpui/management/commands/update_dbt_binary_path.py

```
@@ -0,0 +1,150 @@
+
+    def handle(self, *args, **options):
+        """Handle the command execution"""
+        org_slug = options["org_slug"]
+        dbt_binary_path = options["dbt_binary_path"]
+        dry_run = options["dry_run"]
+
+        # Validate organization exists
+        try:
+            org = Org.objects.get(slug=org_slug)
+        except Org.DoesNotExist:
+            raise CommandError(f"Organization with slug '{org_slug}' does not exist")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)